spaceplesiosaur/refactor-tractor
# MERGE THIS SECOND Form styling

@MattTuring @ckoga 

#### What's this PR do?

This adds some styling to the form - I put it all in it's own .scss document.  I added some colors to the buttons so that we have a good mixin, and I centered the inputs.

I also added labels to the inputs for accessibility reasons.

#### Where should the reviewer start?

Look over the new _form.scss file to see what I did.  Scripts was also updated, start at line 391.

#### How should this be manually tested?

You can pull it down and check to make sure the styling looks good on your computer.  Do a quick check that accessibility is still at 100%, and that you can tab through the whole app.

#### Any background context you want to provide?

This should be merged after you merge the Promise.all's, that code accidentally got added in here.

#### What are the relevant tickets?

https://github.com/spaceplesiosaur/refactor-tractor/issues/47
https://github.com/spaceplesiosaur/refactor-tractor/issues/68
https://github.com/spaceplesiosaur/refactor-tractor/issues/72

#### Screenshots (if appropriate)

<img width="470" alt="Screen Shot 2019-10-28 at 1 59 38 PM" src="https://user-images.githubusercontent.com/24462031/67713306-3535cf00-f98b-11e9-81d7-c0b0a32234f6.png">
